main.go

```diff
@@ -222,7 +223,17 @@ func configureLogger(clogger zap.Logger, config server.Config) (zap.Logger, zap.
 
 func dbConnect(multiLogger zap.Logger, dsns []string) *sql.DB {
 	// TODO config database pooling
-	db, err := sql.Open("postgres", "postgresql://"+dsns[0]+"/nakama?sslmode=disable")
+	rawurl := fmt.Sprintf("postgresql://%s?sslmode=disable", dsns[0])
+	url, err := url.Parse(rawurl)
+	if err != nil {
+		multiLogger.Fatal("Bad connection URL", zap.Error(err))
+	}
+
+	if len(url.Path) < 1 {
+		url.Path = "/nakama"
```

Comment:
does this need to include the `/` - earlier on in the migration this wasn't needed?